filebeat/README.md

```diff
@@ -66,6 +66,7 @@ helm install --name filebeat elastic/filebeat --set imageTag=7.3.0
 | `tolerations`            | Configurable [tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)                                                                                                                                                                         | `[]`                                                                                                                      |
 | `nodeSelector`           | Configurable [nodeSelector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)                                                                                                                                                                | `{}`                                                                                                                      |
 | `affinity`               | Configurable [affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)                                                                                                                                                      | `{}`                                                                                                                      |
+| `priorityClassName`      | The [name of the PriorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass). No default is supplied as the PriorityClass must be created first.                                                                               | ``                                                                                                                        |
```

Comment:
This should be an empty string not an empty value (doesn't really matter though).

```suggestion
| `priorityClassName`      | The [name of the PriorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass). No default is supplied as the PriorityClass must be created first.                                                                               | `""`                                                                                                                        |
```

Reply:
I just went with how this was configured in the other charts that also have `priorityClassName` available; I'm happy to change it if you wish?

Reply:
Go for it! If you could fix it in the other charts too that would be great :)

Reply:
I'll handle it in a separate PR.